source/utils/bpcmp/CMakeLists.txt

```diff
@@ -1,7 +1,6 @@
 configure_file(bpcmp.py ${PROJECT_BINARY_DIR}/bin/bpcmp.py COPYONLY)
 
 install(PROGRAMS bpcmp.py
-  RENAME bpcmp
```

Comment:
Is this intended?

Reply:
Yes, I found it weird for bpcmp to loose the py extension in the install bin folder. If there is any reason for this to happen I can re-add the line but if I was a user I would prefer using the python extension.

Reply:
Sounds good, it is customary to omit the extension in installed scripts since uses do not care which lang it was the script written for them it is just another command. However, if the rest of the scripts are `.py` I say make this also `.py` since what we def do not want are some with `.py` and some without it.